lightning-tag/detailed-reference.md

```diff
@@ -1,3 +1,185 @@
+`getAudiences([limit])` | Array | `limit` - optional, integer, defaults to no limit | Array of either targeting codes as strings or audience ids as integers, based on your established tag settings. Example: `["aud1","aud2"]` or `[123,789]`
+
+#### Local Storage Usage
+
+As noted above in the `config` object created at script setup, there is a parameter named `audienceLocalStorage`. If passed, when targeting is completed, the audiences are written out to the browser's localStorage.
+
+Value | Result
+:---: | ------
+`false` | This is the default, and audiences are not written to browser localStorage.
+`true` |  Lightning Tag writes audiences to localStorage under the key `lotame_CLIENT_ID_auds` (where CLIENT_ID is your Lotame clientId)
+`'string'` | Passing a string tells Lightning Tag to write the audiences to localStorage under the key of the string you pass. (i.e. `audienceLocalStorage: 'myLotameAudiences'`)
+
+Audiences are written out as a string of comma-delimited targeting codes or audience IDs, based on your established tag settings. Example: `"aud1,aud2"` or `"123,789"`
+
+#### Tag Ready Callback
+
+The `onTagReady` callback fires when the Lotame Lightning Tag interface is ready to be used. Which includes below described methods such as `collect` and `page`.
+
+```javascript
+onTagReady: function(namespace) {
+  // Lightning Tag script is ready to be used
+},
+```
+
+?> This event does not confirm audiences are available, that is the `onProfileReady` callback described above in this document.
+
+#### Auto Run Flag
+
+The `autoRun` boolean is defaulted `true` and runs the Lotame data collection and targeting retrieval is run immediately after the script is loaded. By default it is `true` so those functions run automatically after the script is loaded. If set to `false` control is left to the client's website to fire data collection and targeting using the `collect()` and `page()` methods described lower on this document.
+
+A use-case for setting this to `false` is if you want to get the customer's consent before enabling the data collection and targeting to run.
+
+### Data Object
+
+The `data` object is used to pass first party-data to the Lotame DMP for use in targeting or other features of the Lotame platform. The object is optionally passed at setup time in the tag input function. It can also be passed post-page load in the `collect({data})` or in the `page({data})` methods that is described further down this page.
+
+The `data` object's parameters are fully described in [Lightning Tag Data Collection](lightning-tag/data-collection.md).
+
+## Lightning Tag Methods
+
+!> To ensure any of the below methods are available, ensure the `onTagReady` callback has returned else the below methods are not available yet.
+
+### collect()
+
+The `collect()` method is available to pass data to your Lotame DMP the Lotame javascript is loaded. An example use-case is tracking events such as user interaction with a video player on your site.
```

Comment:
... the Lotame DMP once the Lotame javascript ... ?

Reply:
I missed 'once'